src/Illuminate/Routing/RoutingServiceProvider.php

```diff
@@ -7,10 +7,11 @@
 use Illuminate\Contracts\View\Factory as ViewFactoryContract;
 use Illuminate\Routing\Contracts\ControllerDispatcher as ControllerDispatcherContract;
 use Illuminate\Support\ServiceProvider;
+use Nyholm\Psr7\Factory\Psr17Factory;
+use Nyholm\Psr7\Response as PsrResponse;
 use Psr\Http\Message\ResponseInterface;
 use Psr\Http\Message\ServerRequestInterface;
-use Symfony\Bridge\PsrHttpMessage\Factory\DiactorosFactory;
-use Zend\Diactoros\Response as PsrResponse;
```

Comment:
We can't just delete this, since anyone who had zend installed but not the symfony factory will have thier app broken. We'd have to delay not looking at zend to 7.x.

Reply:
Ah - great point. Update incoming.